src/module/cache-container/config/tasks/view/edit-script-modal.html

```diff
@@ -38,54 +48,74 @@ <h4 class="modal-title" id="edit_script.title"><span ng-if="!ctrl.editing" trans
+          </i>
         </div>
       </div>
-      <!--------------------------------------------------------------------------------------------------------------->
+
       <div class="form-group" ng-show="!ctrl.editing">
         <label class="col-md-3 control-label">* <span translate>EXECUTION_MODE</span>:</label>
 
         <div class="col-md-8">
-          <select id-generator="edit.exec_mode" class="form-control" name="taskExecutionMode" ng-model="ctrl.task.mode" ng-required="true">
+          <select id-generator="edit.exec_mode"
+              class="form-control"
+              name="taskExecutionMode"
+              ng-model="ctrl.task.mode"
+              ng-required="true">
             <option value="local">Local</option>
             <option value="distributed">Distributed</option>
           </select>
         </div>
         <div class="col-md-1 pull-left">
           <i class="fa fa-info-circle ng-scope" tooltip-animation="true" tooltip-placement="left-bottom"
-             tooltip-animation="true" tooltip-placement="left-bottom" uib-tooltip="Script execution mode"></i>
+            tooltip-animation="true" tooltip-placement="left-bottom" uib-tooltip="Script execution mode"></i>
         </div>
       </div>
-      <!--------------------------------------------------------------------------------------------------------------->
+
       <div class="form-group" ng-show="!ctrl.editing">
         <label class="col-md-3 control-label"><span translate>SECURITY_ROLE</span>:</label>
 
         <div class="col-md-8">
-          <input id-generator="edit.roles" class="form-control" type="text" name="taskRole" ng-model="ctrl.task.role" ng-required="false">
+          <input id-generator="edit.roles"
+              class="form-control"
+              type="text"
+              name="taskRole"
+              ng-model="ctrl.task.role"
+              ng-required="false"/>
         </div>
         <div class="col-md-1 pull-left">
           <i class="fa fa-info-circle ng-scope" tooltip-animation="true" tooltip-placement="left-bottom"
-             uib-tooltip="A specific role which is required to execute the script"></i>
+            uib-tooltip="A specific role which is required to execute the script"></i>
         </div>
       </div>
-      <!--------------------------------------------------------------------------------------------------------------->
+
       <div class="form-group">
         <label class="col-md-3 control-label">* <span translate>SCRIPT_BODY</span>:</label>
 
         <div class="col-md-8">
-          <textarea id-generator="edit.body.text" class="form-control" rows="15" ng-model="ctrl.task.body" name="taskBody" ng-required="true"></textarea>
+          <textarea id-generator="edit.body.text"
+              class="form-control"
+              rows="15"
+              ng-model="ctrl.task.body"
+              name="taskBody"
+              ng-required="true">
+          </textarea>
```

Comment:
Are the changes in this file purely cosmetic? If so I think it's best to avoid as it makes performing diffs harder as it means we have to spend time figuring out whether it is a logical or cosmetic change.